src/main/java/com/conveyal/gtfs/error/NewGTFSErrorType.java

```diff
@@ -45,22 +45,25 @@ public enum NewGTFSErrorType {
     FLEX_FORBIDDEN_PRIOR_NOTICE_START_DAY_FOR_BOOKING_TYPE(Priority.HIGH, "It is forbidden to define a prior notice start day for booking type 0 (Real time booking)."),
     FLEX_FORBIDDEN_PRIOR_NOTICE_START_DAY(Priority.HIGH, "It is forbidden to define a prior notice start day for booking type 1 (Up to same-day booking with advance notice) and if prior notice duration max is defined."),
     FLEX_FORBIDDEN_PRIOR_START_TIME(Priority.HIGH, "It is forbidden to define a prior notice start time if prior notice start day is not defined."),
-    FLEX_REQUIRED_END_PICKUP_DROP_OFF_WINDOW(Priority.HIGH, "End pickup drop off window is required if a stop id refers to a stop area or location."),
+    FLEX_FORBIDDEN_STOP_ID(Priority.HIGH, "It is forbidden to define a stop id if a location group id or location id are defined."),
+    FLEX_FORBIDDEN_ROUTE_CONTINUOUS_DROP_OFF(Priority.HIGH, "It is forbidden to define a continuous drop off if start/end pick up drop off windows are defined for any trip of this route."),
+    FLEX_FORBIDDEN_ROUTE_CONTINUOUS_PICKUP(Priority.HIGH, "It is forbidden to define a continuous pick up if start/end pick up drop off windows are defined for any trip of this route."),
+    FLEX_REQUIRED_END_PICKUP_DROP_OFF_WINDOW(Priority.HIGH, "End pickup drop off window is required if a location group id, location id or start pickup drop off window is defined."),
     FLEX_REQUIRED_PRIOR_NOTICE_DURATION_MIN(Priority.HIGH, "Prior notice duration min is required for booking type 1 (Up to same-day booking with advance notice)."),
     FLEX_REQUIRED_PRIOR_NOTICE_LAST_DAY(Priority.HIGH, "Prior notice last day is required if a stop id refers to a stop area or location."),
     FLEX_REQUIRED_PRIOR_NOTICE_START_TIME(Priority.HIGH, "Prior notice start time is required if prior notice start day is defined."),
-    FLEX_REQUIRED_START_PICKUP_DROP_OFF_WINDOW(Priority.HIGH, "Start pickup drop off window is required if stop id refers to a stop area or location."),
-    FLEX_SAFE_FACTORS_EXCEEDED(Priority.HIGH, "Safe duration factor + safe duration offset should be greater than mean duration factor + mean duration offset."),
+    FLEX_REQUIRED_STOP_ID(Priority.HIGH, "A stop id is required if a location group id or location id are not defined."),
```

Comment:
```suggestion
    FLEX_REQUIRED_STOP_ID(Priority.HIGH, "A stop id is required if both location group id and location id are not defined."),
```
and do you want to use the exact field names?